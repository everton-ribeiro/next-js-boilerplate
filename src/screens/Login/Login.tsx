import React from "react";

import Title from "@components/Title";

export const Login = (): JSX.Element => {
  return <Title>Home</Title>;
};
