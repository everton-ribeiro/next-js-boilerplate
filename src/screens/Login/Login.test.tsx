/* eslint-disable import/no-extraneous-dependencies */
import React from "react";
import { render, screen } from "@testing-library/react";

import Login from ".";

describe("<Login />", () => {
  it("should render the heading", () => {
    const { container } = render(<Login />);
    expect(screen.getByRole("heading", { name: /Home/i })).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
