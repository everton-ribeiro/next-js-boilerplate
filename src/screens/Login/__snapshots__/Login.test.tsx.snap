// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> should render the heading 1`] = `
.c0 {
  font-size: 50px;
  color: #ff0000;
}

<h1
  class="c0"
>
  Home
</h1>
`;
